src/main/java/com/groupseven/sysc4806project/Book.java

```diff
@@ -10,6 +10,8 @@ public class Book {
     private int id;
     private int inventory;
 
+    private int orderAmount;
```

Comment:
I think order amount should be on the shopping cart, not the book. A book is used as a listing for the store and there should only be one version of it in the system for any given title.

Reply:
I did it this way because when ordering books, the order amount is unique to each book as the user can specify how many they would like to order, wether 1 copy of said book or multiple. I don't how this would work in shopping cart as it's unique to each book which is why I created it in the Book class

Reply:
The way you currently have it will make it so if multiple users have the book in their cart and one user updates their order amount, it will update the order amount for all users as the book object is shared. Instead of storing the cart as a list, maybe it could be a `Map<Integer, Integer>` mapping book IDs to the order amount. Would require other changes but I don't think it would be too difficult